source/gui/__init__.py

```diff
@@ -75,6 +77,13 @@ def __init__(self):
 				self.Show()
 				self.Hide()
 
+	def Destroy(self):

```

Comment:
Note, if wx calls destroy automatically for some reason this won't be called (see https://github.com/wxWidgets/Phoenix/issues/630). This has tripped us up in the past. It might be better to move the `sysTrayIcon` destruction into an `onDestroy` event handler and register for that event.

Reply:
Hmm, sadly this can't be done for the destruction of the `wx.Menu` from `sysTrayIcon` as `sysTrayIcon` doesn't have a destruction event handler because it's not a window. I hope that also means that `wx` also won't be destroying it automatically.

I've fixed this for this function though.

Reply:
Ah yep, that makes sense. It's a shame there are so many exceptions to learn about with wx!